src/test/regress/expected/alter_table_set_access_method.out

```diff
@@ -356,6 +356,7 @@ SELECT create_reference_table('table_type_ref');
 (1 row)
 
 CREATE TABLE table_type_citus_local(a INT);
+SET citus.enable_local_reference_table_foreign_keys TO OFF;
```

Comment:
I guess we should remove these `SET citus.enable_local_reference_table_foreign_keys TO OFF;` calls so that we can check the CI easier